templates/hotgraphic.hbs

```diff
@@ -65,11 +74,16 @@
           <div class="graphic-attribution">{{{_graphic.attribution}}}</div>
         {{/if}}
         {{#each _items}}
-          <button class="base hotgraphic-graphic-pin component-item-text-color item-{{@index}} {{#if _isVisited}}visited{{/if}}" data-id="item-{{@index}}" style="top:{{{_top}}}%; left:{{{_left}}}%;" role="button" aria-label="Item {{numbers @index}}. {{title}}.{{#if _isVisited}} {{../../_globals._accessibility._ariaLabels.visited}}.{{/if}}">
-            <div class="hotgraphic-graphic-pin-icon component-item-color icon icon-pin item-{{@index}}"></div>
+          <button class="base hotgraphic-graphic-pin component-item-text-color item-{{@index}} {{#if _isVisited}}visited{{/if}} {{#compare ../_pinStyle '=' 'numbers' }}{{#unless _iconClass}}number{{/unless}}{{/compare}}" data-id="item-{{@index}}" style="top:{{{_top}}}%; left:{{{_left}}}%;" role="button" aria-label="Item {{numbers @index}}. {{title}}.{{#if _isVisited}} {{../../_globals._accessibility._ariaLabels.visited}}.{{/if}}">
+            {{#compare ../_pinStyle '=' 'numbers' }}
+            <div class="{{#if _iconClass}}hotgraphic-graphic-pin-icon icon {{_iconClass}}{{else}}hotgraphic-graphic-number-icon{{/if}} component-item-color item-{{@index}}">{{#unless _iconClass}}{{inc @index}}{{/unless}}</div>
```

Comment:
People may want to have the option of having a combination of numbers and icons for example 1-4 and then a star at the end.